examples/backward_modes.py

```diff
@@ -128,6 +128,22 @@ def quad_error_fn(optim_vars, aux_vars):
 print(da_dx.numpy())
 
 
+# We can also compute the direct loss minimization gradient.
+updated_inputs, info = theseus_optim.forward(
```

Comment:
Example looks great! Probably also add to unit tests like [here](https://github.com/facebookresearch/theseus/blob/8741e372d07252e1a14ab0fdc7afebcf23f0557e/theseus/optimizer/nonlinear/tests/test_backwards.py#L65).

Reply:
Will do.